lxd/auth/drivers/openfga.go

```diff
@@ -247,6 +247,7 @@ func (e *embeddedOpenFGA) CheckPermission(ctx context.Context, entityURL *api.UR
 		// If we have a not found error from the underlying OpenFGADatastore we should mask it to make requests consistent.
 		// (all not found errors returned before an access control decision is made are masked to prevent discovery).
 		if api.StatusErrorCheck(err, http.StatusNotFound) {
+			l.Info("Entity not found", logger.Ctx{"http_code": http.StatusNotFound})
```

Comment:
should these be warnings? They shouldnt happen normally right?

Reply:
This happens normally in many cases. For example with certificates, permissions are checked before we've attempted to access the certificate from the database. So if someone calls `PUT /1.0/certificates/not-a-certificate` we'll hit this code path.